[Darcy] Rare crash

I've seen it once before, I think.

https://github.com/rems-project/cn/actions/runs/23474044637/job/68302955253#step:11:475

```
Using seed: dba77811a9f13289

Testing slf_sized_stack::IntList_cons: 100 runs
PASSED
Testing slf_sized_stack::IntList_length: 100 runs
PASSED
Testing slf_sized_stack::sizeOf: 100 runs
PASSED
Testing slf_sized_stack::push: 100 runs
PASSED
Testing slf_sized_stack::pop: 100 runs, 11 discards
Capturing coverage data from .
geninfo cmd: '/usr/bin/geninfo . --output-filename coverage.info --gcov-tool gcov --base-directory . --memory 0'
Found gcov version: 13.3.0
Using intermediate gcov format
Writing temporary data to /tmp/geninfo_datyvih
Scanning . for .gcda files ...
Failed to collect coverage.src/slf_sized_stack.pass.c:155:14: warning: deprecated keyword 'Owned', use 'RW' instead
    p->data = tail;
             ^~~~~ 

Unexpected solver response in get_expr - expected single element list: (error "line 7 column 12: unknown constant _sym_1")
tests.out: ../../src/cn-smt/from_smt.c:25: unexpected_solver_response: Assertion `false' failed.
/tmp/bennet.D68E/run_tests.sh: line 37: 45340 Aborted                 (core dumped) ./tests.out --print-seed --input-timeout 1000 --progress-level function --max-generator-size 16 --print-backtrack-info --print-satisfaction-info --smt-pruning-at-runtime --smt-skewing sized
geninfo: ERROR: no .gcda files found in .
	(use "geninfo --ignore-errors empty ..." to bypass this error)
```

Another one:
```
         config: -I/home/runner/.opam/4.14.1/lib/cerberus-lib/runtime/libc/include/posix --input-timeout=1000 --progress-level=function --sanitize=address,undefined --allow-split-magic-comments --max-generator-size=16 --print-seed --symbolic --coverage --print-backtrack-info --print-satisfaction-info --smt-pruning-at-runtime --build-tool=make
Using temporary directory: /tmp/bennet.8BDC
cc -c -o slf_sized_stack.pass.test.o slf_sized_stack.pass.test.c -g -I/home/runner/.opam/4.14.1/lib/cn/runtime/include/   -Werror -Wall -Wno-unused-value -Wno-unused-variable -Wno-unused-but-set-variable -Wno-unused-label -Wno-unused-function -fsanitize=address,undefined --coverage
cc -c -o slf_sized_stack.pass.exec.o slf_sized_stack.pass.exec.c -g -I/home/runner/.opam/4.14.1/lib/cn/runtime/include/   -Werror -Wall -Wno-unused-value -Wno-unused-variable -Wno-unused-but-set-variable -Wno-unused-label -Wno-unused-function -fsanitize=address,undefined --coverage
cc -o tests.out slf_sized_stack.pass.test.o slf_sized_stack.pass.exec.o /home/runner/.opam/4.14.1/lib/cn/runtime/libcn_test.a /home/runner/.opam/4.14.1/lib/cn/runtime/libbennet.a "/home/runner/.opam/4.14.1/lib/cn/runtime/libcn_smt.a" /home/runner/.opam/4.14.1/lib/cn/runtime/libcn_replica.a /home/runner/.opam/4.14.1/lib/cn/runtime/libcn_exec.a -g -I/home/runner/.opam/4.14.1/lib/cn/runtime/include/   -Werror -Wall -Wno-unused-value -Wno-unused-variable -Wno-unused-but-set-variable -Wno-unused-label -Wno-unused-function -fsanitize=address,undefined --coverage
./tests.out --print-seed --input-timeout 1000 --progress-level function --max-generator-size 16 --print-backtrack-info --print-satisfaction-info --smt-pruning-at-runtime --smt-skewing sized
Using seed: 745d38713706daae

PASSED

PASSED
Testing slf_sized_stack::IntList_cons: 100 runs
PASSED
Testing slf_sized_stack::IntList_length: 100 runs
PASSED
Testing slf_sized_stack::sizeOf: 100 runs
PASSED
Testing slf_sized_stack::push: 100 runs
PASSED
Testing slf_sized_stack::pop: 100 runs, 11 discardssrc/slf_sized_stack.pass.c:155:14: warning: deprecated keyword 'Owned', use 'RW' instead
    p->data = tail;
             ^~~~~ 

Unexpected solver response in get_expr - expected single element list: (error "line 7 column 12: unknown constant _sym_1")
tests.out: ../../src/cn-smt/from_smt.c:25: unexpected_solver_response: Assertion `false' failed.
```

--- a/lib/testGeneration/bennet/symbolic/harness.ml
+++ b/lib/testGeneration/bennet/symbolic/harness.ml
@@ -233,7 +233,7 @@ module Make (AD : Domain.T) = struct
     ^/^ state_init
     ^/^ vars_decl
     ^/^ !^"int attempts = 0;"
-    ^/^ !^"enum cn_smt_solver_result result;"
+    ^/^ !^"enum cn_smt_solver_result result = CN_SOLVER_UNKNOWN;"
     ^/^ (!^"do"
          ^^^ braces
                (hardline
@@ -247,6 +247,7 @@ module Make (AD : Domain.T) = struct
                        ^/^ !^"branch_history_clear(&branch_hist);"
                        ^/^ !^"bennet_failure_reset();"
                        ^/^ !^"attempts++;"
+                       ^/^ !^"result = CN_SOLVER_UNKNOWN;"
                        ^/^ !^"continue;")
                 ^/^ !^"/* Gather constraints */"
                 ^/^ gather_section_with_timing

